src/gateways/CachedRegistrationGateway.ts

```diff
@@ -1,4 +0,0 @@
-export interface CachedRegistrationGateway {
-  deleteUse: (submissionId, useIndex) => Promise<void>;
-  createEmptyUse: (submissionId) => Promise<void>;
-}
```

Comment:
In keeping with thinking of `<Entity>Gateway`, renamed this interface to `DraftRegistrationGateway`.